packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/diagram/NodeStylePropertiesConfigurer.java

```diff
@@ -89,13 +89,16 @@ public class NodeStylePropertiesConfigurer implements IPropertiesDescriptionRegi
 
     private final IObjectService objectService;
 
+    private final Function<VariableManager, String> semanticTargetIdProvider;
+
     public NodeStylePropertiesConfigurer(ICustomImageMetadataSearchService customImageSearchService, IValidationService validationService,
             List<IParametricSVGImageRegistry> parametricSVGImageRegistries, AQLTextfieldCustomizer aqlTextfieldCustomizer, IObjectService objectService) {
         this.validationService = Objects.requireNonNull(validationService);
         this.customImageSearchService = Objects.requireNonNull(customImageSearchService);
         this.parametricSVGImageRegistries = parametricSVGImageRegistries;
         this.aqlTextfieldCustomizer = Objects.requireNonNull(aqlTextfieldCustomizer);
         this.objectService = Objects.requireNonNull(objectService);
+        this.semanticTargetIdProvider = variableManager -> variableManager.get(VariableManager.SELF, Object.class).map(objectService::getId).orElse(null);
```

Comment:
I don't have a strong opinion on the matter, but for semanticElementsProvider we declared its value directly at initialization and not in the constructor.